src/PrestaShopBundle/Form/Admin/Sell/Product/Details/ReferencesType.php

```diff
@@ -79,11 +79,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('ean_13', TextType::class, [
                 'required' => false,
-                'label' => $this->trans('EAN-13 or JAN barcode', 'Admin.Catalog.Feature'),
+                'label' => $this->trans('GTIN (EAN, JA, ITF or UCC code)', 'Admin.Catalog.Feature'),
                 'label_help_box' => $this->trans('This type of product code is specific to Europe and Japan, but is widely used internationally. It is a superset of the UPC code: all products marked with an EAN will be accepted in North America.', 'Admin.Catalog.Help'),
                 'constraints' => [
-                    new TypedRegex(TypedRegex::TYPE_EAN_13),
-                    new Length(['max' => Ean13::MAX_LENGTH]),
+                    new TypedRegex(TypedRegex::TYPE_GTIN),
+                    new Length(['max' => Gtin::MAX_LENGTH]),
```

Comment:
There is the same thing here https://github.com/PrestaShop/PrestaShop/blob/13899326776db34996cca645c34d0f1fbf9fc585/src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/BulkCombinationReferencesType.php#L85